modules/google-analytics/wp-google-analytics.php

```diff
@@ -50,7 +50,9 @@ class Jetpack_Google_Analytics {
 	 * @return void
 	 */
 	private function __construct() {
-		if ( Jetpack_Google_Analytics_Options::enhanced_ecommerce_tracking_is_enabled() ) {
+		// At this time, we only leverage universal analytics when enhanced ecommerce is selected and WooCommerce is active.
+		// Otherwise, don't bother emitting the tracking ID or fetching analytics.js
+		if ( Jetpack_Google_Analytics_Options::enhanced_ecommerce_tracking_is_enabled() && class_exists( 'WooCommerce' ) ) {
```

Comment:
This is extremely nitpicky, but sharing to help frame something to keep in mind. With an `&&` operation, it checks left-to-right and fails on the first failed condition—so the `enhanced_ecommerce_tracking_is_enabled`, then the `class_exists` check. The `class_exists` check is pretty cheap, but `enhanced_ecommerce_tracking_is_enabled` eventually calls `get_option`, which may or may not be a db hit (didn't look to see if the option is autoloaded or not).

In this case, it probably isn't a huge thing, but for the sake of performance, thinking about the order things are checked can help you out.

Reply:
That makes sense to me! Thanks Kraft 👍

I've pushed a new commit to reorder the condition positions